README.in

```diff
@@ -0,0 +1,69 @@
+# `torch_ttnn` Module
```

Comment:
* We should find a way to strongly indicate that Readme.md is generated
* Will be easier to edit this file if it has .md extension. Just a thought. Maybe move to docs/ folder or any other ideas

Reply:
- I inject a comment to README.md from the collect script so anyone who opens README.md should hopefully see that they need to edit README.md.in instead.
- I will change it to .md.in and I'll move it to docs.